api/CSSPseudoElement.json

```diff
@@ -33,39 +33,17 @@
                   "name": "dom.animations-api.getAnimations.enabled"
                 }
               ]
-            },
-            {
-              "version_added": "47",
-              "version_removed": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.core.enabled"
-                }
-              ]
-            }
-          ],
-          "firefox_android": [
-            {
-              "version_added": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.getAnimations.enabled"
-                }
-              ]
-            },
-            {
-              "version_added": "47",
-              "version_removed": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.core.enabled"
-                }
-              ]
             }
           ],
+          "firefox_android": {
+            "version_added": "63",
```

Comment:
Is the Android data just out of sync with desktop here? Why no "dom.css_pseudo_element.enabled" entry?

Reply:
The changes to add the new flag are in #5784.  My guess is it's just a desync between Firefox and Fenix, but of course, since Fenix doesn't support flags at all anymore, we should probably add a `"version_removed": "79"` statement?

Reply:
Yes, but just putting an end to the flags statement wouldn't be correct if the data is out of sync. Rather there should be another (newer) entry that ends at 79.

Reply:
Sorry, I realize that I poorly worded my reply (I'm horrible at explanations, haha)... 
 When I said "desync", what I was really trying to say was that data was added for Firefox Desktop, but not for Firefox Android, as Fenix was still in development and there were no FxA releases.  Since FxA doesn't have any flags any longer, `dom.css_pseudo_element.enabled` was never introduced to begin with.

Reply:
If Firefox for Android 79 is when flags were discontinued, then there would be a 75-79 entry for "dom.css_pseudo_element.enabled", right?

Reply:
There was no Firefox Android releases between 68 and 79 as a part of the development of the new Fenix edition, so Firefox Android 75-78 never existed.

Reply:
Aha, right!

Reply:
So then I guess a `"version_removed": "79"` line is all that's missing, as you suggested.